docs/src/lib/binary_functions.md

```diff
@@ -122,7 +122,7 @@ issubset
 ⊆(::AbstractSingleton, ::AbstractSingleton, ::Bool=false)
 ⊆(::Ball2, ::Ball2, ::Bool=false)
 ⊆(::Union{Ball2, Ballp}, ::AbstractSingleton, ::Bool=false)
-⊆(::LineSegment, ::ConvexSet, ::Bool=false)
+⊆(::LineSegment{N}, ::ST, ::Bool=false) where {ST<:LazySet{N}}
```

Comment:
```suggestion
⊆(::LineSegment{N}, ::ST, ::Bool=false) where {N, ST<:LazySet{N}}
```

Reply:
I think in this case it might be better
```julia
⊆(::LineSegment, ::ST, ::Bool=false) where {ST<:LazySet}
```

unused type parameters can make the function slower, see ⊆(::LineSegment{N}, ::ST, ::Bool=false) where {N, ST<:LazySet{N}}

Reply:
since the conversation was resolved about while I was writing and not sure how notifications work in that case, pinging @schillic @mforets to make sure the comment doesn't go unnoticed

Reply:
Yes, I just wanted to fix the build. But now there is an ambiguity, so @mforets maybe you can remove the type parameter.